### DISCUSSION TITLE
Popover does not have a showDelay prop when onHover is set

### DISCUSSION THREAD

**Discussion by JackSharpe94** [microsoft/fluentui#31015] [Ideas] [answered]
If I have a popover that shows on hover. Currently there is not a showDelay prop only a mouseLeaveDelay prop.

It would be useful if this prop was present, the current solution I have which works is to control the onMouseEnter and onMouseLeave on the element inside of the trigger and set a timeout on some state for the show prop:
```
    const timeoutID = useRef<any | undefined>(undefined);
    const [open, setOpen] = useState(false);

<Popover open={open} withArrow={true} positioning={TaskPosition}>
            <PopoverTrigger>
                  <div
                         onMouseEnter={() => (timeoutID.current = setTimeout(() => setOpen(true), 1000))}
                         onMouseLeave={() => {
                              clearTimeout(timeoutID.current);
                              setOpen(false);
                          }}>                   
                         ...
                 </div>
              </PopoverTrigger>
              <PopoverSurface>
                     <HoverCard} />
              </PopoverSurface>
</Popover>
```

It would be ideal to have the ability to do something like the following on the Popover component: 

`<Popover showDelay={1000} onHover={true}>`

open to suggestions on better approaches

**Comment by bsunderhus** [comment, marked as answer]
Thanks for the discussion @JackSharpe94, this sounds like a Feature request! would be better if you could open an issue with a Feature request instead! Here's the link to open a [feature request](https://github.com/microsoft/fluentui/issues/new?assignees=&labels=Type%3A+Feature%2CNeeds%3A+Triage+%F0%9F%94%8D&projects=&template=feature-request.yml&title=%5BFeature%5D%3A+)

**Reply by JackSharpe94** [reply]
Thank you for the quick response @bsunderhus , will mark as complete and move to request as suggested.
